core/driver/volume.c

```diff
@@ -466,6 +468,45 @@ REDSTATUS RedVolTransact(void)
 
     return ret;
 }
+
+/** @brief Rollback to the previous transaction point.
+
+    @return A negated ::REDSTATUS code indicating the operation result.
+
+    @retval 0           Operation was successful.
+    @retval -RED_EBUSY  A discarded block is still referenced.
```

Comment:
Same comments about `-RED_EBUSY` and `-RED_EIO`.

Reply:
OK